test/data/allele_test.json

```diff
@@ -58,6 +58,33 @@
         ],
         "internal": false
       }
+    },
+    {
+      "curie": "WB:WBTransgene00023682",
+      "taxon_curie": "NCBITaxon:6239",
+      "created_by_curie": "WB:curator",
+      "updated_by_curie": "WB:curator",
+      "internal": false,
+      "allele_symbol_dto": {
+        "name_type_name": "nomenclature_symbol",
+        "format_text": "vsIs183",
+        "display_text": "vsIs183",
+        "internal": false
+      },
+      "references": [
+        {
+          "curie": "WB:WBPaper00050437"
+        },
+        {
+          "curie": "WB:WBPaper00054725"
+        },
+        {
+          "curie": "WB:WBPaper00056478"
+        },
+        {
+          "curie": "WB:WBPaper00057105"
+        }
+      ]
```

Comment:
This should be `reference_curies` and then just a list of strings (not objects with curie properties)

Reply:
@markquintontulloch I've made the corrections